examples/plot_gallery_version.py

```diff
@@ -33,17 +35,26 @@ def bump(a):
             bump(a[:, i])
     return np.abs(a)
 
-with plt.xkcd():
 
-    fig = plt.figure()
+fig = plt.figure()
+d = layers(3, 100)
+try:
+    with plt.xkcd():
+        plt.stackplot(range(100), d.T, baseline='wiggle')
+        plt.annotate(
+            'Introducing Sphinx-Gallery ' + sphinxgallery.__version__,
+            xy=(12, 4), arrowprops=dict(arrowstyle='->'), xytext=(15, -4))
+except Exception:
```

Comment:
Yes this uses the matplotlib cartoon output which is no available in matplotlib 1.1 which is ubuntu 12.04 default. So travis was failing this build